src/builtin.ts

```diff
@@ -138,15 +138,26 @@ for (const name of MATH_KEYS) {
   MATH_WRAPPERS[name as string] = wrapper;
 }
 
-function atan2(x: Interval, y: Interval) {
+function atan2(y: Interval, x: Interval) {
   return new Interval(
-    TimeMonzo.fromValue(Math.atan2(x.valueOf(), y.valueOf())),
+    TimeMonzo.fromValue(Math.atan2(y.valueOf(), x.valueOf())),
     'linear'
   );
 }
-atan2.__doc__ = 'Calculate atan2(x, y).';
+atan2.__doc__ = 'Calculate atan2(y, x) which is the angle between (1, 0) and (x, y), chosen to lie in (−π; π], positive anticlockwise.';
 atan2.__node__ = builtinNode(atan2);
 
+// Equivalent to atan2 but with swapped arguments.
+// Rationale is that atanXY(x, y) = log(x + i * y), x and y now coordinates.
+function atanXY(x: Interval, y: Interval) {
+  return new Interval(
```

Comment:
Just call `atan2` defined above with swapped arguments. Leaves less room for error if there's more domain finagling in the future or something.

Reply:
Yeah I don’t know why I left it that way. Ah I remember now, because I wasn’t sure if `atan2` is to live.